src/makeRobot.js

```diff
@@ -38,7 +38,58 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    'name': name,
+    'version': version,
+    'wheels': wheels,
```

Comment:
Если имя свойства === названию переменной, можно использовать сокращенную запись:
```suggestion
    name,
    version,
    wheels,
```